pkg/envoy/container/defaults/defaults.go

```diff
@@ -0,0 +1,68 @@
+package defaults
+
+import (
+	"strings"
+
+	"github.com/3scale-ops/marin3r/pkg/envoy"
+	"github.com/3scale-ops/marin3r/pkg/version"
+)
+
+const (
+	// common defaults
+	Image                           string = "envoyproxy/envoy:v1.16.0"
+	EnvoyConfigBasePath             string = "/etc/envoy/bootstrap"
+	EnvoyConfigFileName             string = "config.json"
+	EnvoyExtraArgs                  string = ""
+	EnvoyTLSBasePath                string = "/etc/envoy/tls/client"
+	EnvoyAPIVersion                 string = string(envoy.APIv2)
+	TlsCertificateSdsSecretFileName string = "tls_certificate_sds_secret.yaml"
+	EnvoyAdminPort                  uint32 = 9901
+	EnvoyAdminAccessLogPath         string = "/dev/null"
+	GracefulShutdownTimeoutSeconds  int64  = 300
+
+	LivenessInitialDelaySeconds int32 = 30
+	LivenessTimeoutSeconds      int32 = 1
+	LivenessPeriodSeconds       int32 = 10
+	LivenessSuccessThreshold    int32 = 1
+	LivenessFailureThreshold    int32 = 10
+
+	ReadinessProbeInitialDelaySeconds int32 = 15
+	ReadinessProbeTimeoutSeconds      int32 = 1
+	ReadinessProbePeriodSeconds       int32 = 5
+	ReadinessProbeSuccessThreshold    int32 = 1
+	ReadinessProbeFailureThreshold    int32 = 1
+
+	// sidecar specific defaults
+	SidecarContainerName        string = "envoy-sidecar"
+	SidecarBootstrapConfigMapV2 string = "envoy-sidecar-bootstrap"
+	SidecarBootstrapConfigMapV3 string = "envoy-sidecar-bootstrap-v3"
+	SidecarConfigVolume         string = "envoy-sidecar-bootstrap"
+	SidecarTLSVolume            string = "envoy-sidecar-tls"
+	SidecarClientCertificate    string = "envoy-sidecar-client-cert"
+
+	// deployment specific defaults
+	DeploymentContainerName        string = "envoy"
+	DeploymentBootstrapConfigMapV2 string = "envoy-bootstrap"
+	DeploymentBootstrapConfigMapV3 string = "envoy-bootstrap-v3"
+	DeploymentConfigVolume         string = "envoy-bootstrap"
+	DeploymentTLSVolume            string = "envoy-tls"
+	DeploymentClientCertificate    string = "envoy-client-cert"
+
+	// shutdown manager defaults
+	ShtdnMgrDefaultImageRegistry      string = "quay.io/3scale/marin3r"
```

Comment:
So the default shutdown-manager imager is the operator image **without specific tag**? I don't know why I thought it was envoy's image

Reply:
No, that is just the repository where the image is. The tag of the image is calculated depending on marin3r's version:

https://github.com/3scale-ops/marin3r/blob/a6d531ba4c44b5c27b2e157d8cc89eea6501cf85/pkg/envoy/container/defaults/defaults.go#L66-L68

Reply:
Ok nice! didnt saw it :P